core/src/main/java/feast/core/job/dataflow/DataflowJobManager.java

```diff
@@ -283,14 +260,17 @@ private ImportOptions getPipelineOptions(
     pipelineOptions.setJobName(jobName);
     pipelineOptions.setFilesToStage(
         detectClassPathResourcesToStage(DataflowRunner.class.getClassLoader()));
-
+    if (pipelineOptions.getTempLocation() == null && sink.getBigqueryConfig() != null) {
+      pipelineOptions.setTempLocation(sink.getBigqueryConfig().getStagingLocation());
```

Comment:
Is there a way to make this more generic (non-BQ specific)?

Reply:
we actually don't need this part at all, since now it's
`@NotBlank public String tempLocation;`